index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Kossi Fioklou Portfolio</title>
+</head>
+<body>
+    <!--This is the section for the header-->
+      <nav id="nav-bar">
```

Comment:
- [ ] Kindly use a `header` tag to wrap all elements here, in this case you used a `nav` tag